app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryPresenter.kt

```diff
@@ -43,6 +56,59 @@ class TokenHistoryPresenter(
         view?.openSellTransactionDetails(transactionId)
     }
 
+    override fun onBridgePendingClaimClicked(transactionId: String) {
+        launch {
+            val bridgeBundle = ethereumInteractor.getClaimBundleById(transactionId) ?: return@launch
```

Comment:
I Suggest to extract to separate class to remove duplication. Something like BridgeHistoryItemClickHandler